README.md

```diff
@@ -1,15 +1,65 @@
 # hc-web-client
 
-Thin wrapper around `rpc-websockets` to enable calling zome functions in Holochain apps installed in a container.
+Thin wrapper around `rpc-websockets` to enable calling zome functions in Holochain apps installed in a conductor. This module aims to fill a similar role to web3.js and allow for connecting to a holochain instance in a variety of scenarios.
+
+## Installation
+
+The module can either be imported and used as part of a compiled web application or used in pure HTML. In the first case it is reccomended to download and install via npm `npm install @holochain/hc-web-client`. It can then be imported or required using
+
+```javascript
+const { connect } = require('@holochain/hc-web-client')
+// or
+import { connect } from '@holochain/hc-web-client'
+```
+
+For the pure HTML case the index.js file in the `dist/` folder must be moved to your project directory and imported using
+```html
+<script type="text/javascript" src="path/to/holoclient.js"></script>
```

Comment:
holochainClient.js

Reply:
This is a good point you raise. From some research I found it is convention to name the output of the build for the browser build as `(module)-(version).(platform).[min].js` so that there can be no confusion around the version.

I updated the package.json to build using that convention and the example/readme to refer to it in that way.